src/modules/service-catalog/ServiceCatalogItemPage.tsx

```diff
@@ -77,44 +96,78 @@ const ButtonWrapper = styled.div`
   align-items: center;
 
   @media (max-width: ${(props) => props.theme.breakpoints.md}) {
-    position: sticky;
+    position: fixed;
     bottom: 0;
+    left: 0;
+    right: 0;
+    width: 100vw;
```

Comment:
Honestly I wouldn't use `position: fixed` here, otherwise the button is going to cover the footer when the page is scrolled down to the bottom. It is better to use `sticky` which is already set on the parent. 
It is a bit difficult to set the width to be 100% of the page without position fixed, but let's check from design if it is really needed.

Reply:
Changed to `sticky`